### DISCUSSION TITLE
Some of the characters are sometimes not sent to the computer when I use the two-step keyboard

### DISCUSSION THREAD

**Discussion by MyNameIsTwiggy** [AppGround-io/bluetooth-keyboard-and-mouse-support#193] [Bugs & Problems]
Hello,

I use the premium version of the app daily with a OnePlus 6t smartphone. I am very happy with the functionality.

However, sometimes I have a problem with the feature that allows you to type the text in the application before sending it to the computer. Sometimes when I click the "send" button, **only certain characters are sent. Most often a letter is repeated several times and part of the sentence is not sent at all**.

For example, the sentence:
_"This is an example sentence that I want to send"_

can give :
_"This iiiiiiiiiiiiiiiiiiiiint to send"_

The problem most often occurs when I haven't used the app for at least a minute. So I think somehow the Bluetooth goes to sleep and has a problem reconnecting when I go back to the app. I have disabled all battery saving options on my phone that could affect the Bluetooth app and services without success.

I sometimes use the app with a Samsung tablet (Galaxy Tab A SM-T510) and I haven't encountered the problem once. In addition, unless I am mistaken, I have not seen this problem mentioned in this discussion space.

I don't know if this information is relevant but I am using Google's Android keyboard (GBoard).

I would be very grateful if you could help me on this subject.

Good day to you.

**Comment by matrxbox** [comment]
I have encountered similar with physical bluetooth keyboards and devices,  particularly when the host device has other bt devices connected.   Since you are only experiencing it with the oneplus, could it be a bt issue?   

Are other bt devices connected to either device?  I find that android favors prioritising the connection for stereo sound which sometimes interferes .
